simulation/src/main/java/com/palantir/dialogue/core/MetricNames.java

```diff
@@ -23,20 +23,27 @@
 import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
 
 final class MetricNames {
+
+    private static final MetricName RESPONSE_CLOSE =
+            MetricName.builder().safeName("responseClose").build();
+    private static final MetricName GLOBAL_RESPONSES =
+            MetricName.builder().safeName("globalResponses").build();
+    private static final MetricName GLOBAL_SERVER_TIME =
+            MetricName.builder().safeName("globalServerTime").build();
+
     /** Counter incremented every time a {@code Response} is closed. */
     static Counter responseClose(TaggedMetricRegistry reg) {
-        return reg.counter(MetricName.builder().safeName("responseClose").build());
+        return reg.counter(RESPONSE_CLOSE);
```

Comment:
This is cleaner than before imo, but we should update the call sites to look up the metric once per simulation rather than per-request.